src/lutf8lib.js

```diff
@@ -160,7 +161,7 @@ const codepoint = function(L) {
     lauxlib.luaL_argcheck(L, pose <= s.length, 3, lua.to_luastring("out of range", true));
 
     if (posi > pose) return 0;  /* empty interval; return no values */
-    if (pose - posi >= llimit.MAX_INT)
+    if (pose - posi >= INT_MAX)
```

Comment:
Again, I think this could be `Number.MAX_SAFE_INTEGER`: all the places INT_MAX gets used seems to be overflow checks.